source/wp-content/themes/wporg-developer-2023/js/autocomplete.js

```diff
@@ -11,7 +11,7 @@
 		return;
 	}
 
-	var form = $( '.awesomeplete-form-wrap > form' );
+	var form = $( 'main form.wp-block-search' );
```

Comment:
https://github.com/WordPress/wporg-developer/blob/trunk/source/wp-content/themes/wporg-developer-2023/inc/autocomplete.php#L40 already takes care of handbook searchbar